nn_sdwan/assets/service_checks.json

```
@@ -0,0 +1,15 @@
+[
+    {
+        "agent_version": "7.33.0",
+        "integration": "nn_sdwan",
```

must match manifest display_name `Netnology SDWAN` 
```suggestion
        "integration": "Netnology SDWAN,
```